packages/lesswrong/lib/collections/comments/fragments.ts

```diff
@@ -78,6 +78,12 @@ export const CommentsList = () => frag`
     originalDialogueId
 
     forumEventId
+    forumEvent {
```

Comment:
Why put this in the comment fragment, rather than make the substituted widget inside the comment do a separate fetch? It seems like that would be a lot simpler (and would prevent comments and posts from having to be aware of any of this).

Reply:
Seems reasonable 👍, this also makes it easier to refetch the agreement % without refechting all the comments